lua/render-markdown/parser/code_block.lua

```diff
@@ -55,6 +55,10 @@ function M.get_width(config, context, info)
     local longest_line = config.left_pad + code_width + config.right_pad
     local width = math.max(longest_line, config.min_width)
     if config.width == 'block' then
+        width = width + config.language_pad
```

Comment:
This width calculation is not quite right and will end up adding right padding when not necessary.

It essentially only works if the language string is the longest line.

For example this config:

```lua
require('render-markdown').setup({
  code = {
    width = 'block',
    language_pad = 2,
  },
})
```

With this markdown:

````
```lua
print('hello')
```
````

Will result in an extra spaces on the right side of the code block.